plutus-metatheory/src/Utils.lagda.md

```diff
@@ -127,7 +138,9 @@ dec2Either : {A : Set} → Dec A → Either (¬ A) A
 dec2Either (yes p) = inj₂ p
 dec2Either (no ¬p) = inj₁ ¬p
 
--- writer monad
+```
+# Writer Monad
+```
```

Comment:
There's some weird alingment below: `where` is on  a new line for `WriterMonad`.

Reply:
I think the comment "We also have a kind called # which is used for sized integers and bytestrings" is outdated now: it's used for type variables which are polymorphic over built-in types now (* is used for polymorphism over general terms),